src/lib/okta.authorize.ts

```diff
@@ -0,0 +1,30 @@
+import { OAUTH_SERVER, oktaJwtVerifier } from "./okta.connect";
+import { NextFunction, Request, Response } from "express";
+import { errorResponse } from "./responses/error.response";
+
+/**
+ * @description - Intercept any request coming from a user and check
+ *                  the given access token integrity.
+ *
+ * @param res - Response
+ * @param next - Continue
+ * @param authorization - Authorization header
+ */
+function oktaAuthorization(res: Response, next: NextFunction, authorization: string): void {
+    // Authorization header OK, let's check the given token.
+    oktaJwtVerifier.verifyAccessToken(authorization, `api://${OAUTH_SERVER}`)
```

Comment:
let's make this function async.
We can then just `await oktaJwtVerifier.verifyAccessToken`. I hope that this method throws an error if auth failed. 

Then this functions caller should call next() if no error was raised, or stop execution and return an HTTP error if it caught the error.